braga/README.md

```diff
@@ -0,0 +1,655 @@
+
+This price is _exceptionally good value-for-money_
+based on our (_extensive_) market research
+and considering all the benefits people will get (_see below for "what's included"_).
+
+#### _Financially Sustainable_ But _Not For Profit_
+
+At €256/person/month the `Braga House` will cover all costs
+and make a _**small surplus**_ which will be
+***re-invested*** into the house & community.
+
+#### _What's Included_?
+
++ A **single bed**<sup>1</sup> in a ***shared room***<sup>2</sup> with:
+  + an orthopedic memory foam mattress (_or alternative of your choice up to €200_)
+  + your **choice** of **pillow** (_because pillows are a v. personal thing and make **all** the difference to sleep quality!_)
+  + **bed linen** (_highest quality_) is **changed weekly** by housekeeper.
+  (_if you prefer a higher frequency inform the "house parent"_)
++ **State-of the Art Kitchen** with every appliance/gadget, utensil and utility
+you could _ever_ need to unleash your ***culinary creativity** if choose to.
+  + **3 Large Fridges** so everyone has space for their food.
+  + **3 Dishwashers** so nobody has to wash crockery!
++ ***Unlimited*** Healthy Vegan Soup (_freshly made each day_)
++ ***Unlimited*** Fresh Fruit & Vegetables (_personal use only, please don't start a restaurant with the House's "free" ingredients!_)
++ ***Unlimited*** Healthy Beverages:
+  + Filtered water (_carefully controlled and high mineral content_)
+  + Tea (_any type/flavour_)
+  + Coffee (_"barista-style" coffee machine for you "**flat white**" lovers_)
+  + Healthy Smoothies (_made with **the best** "Quiet One" VitaMix **blender**_)
++ _**All Bills** Included_:
+  + **Cleaning service** (_daily_)
+  + **Laundry service** (_each person will have their own laundry bag,
+    when yours is full simply drop it off in the utility room in the morning
+    and your fresh & folded laundry will be available for collection the next day_)
+  + ***Unlimited*** 100Mb Fibre Internet (_**Two** "Paired" Low-latency Fibre-to-the-House **Connections** and the highest speed routers available_)
+  + 100% Renewable Electricity (_Zero Fossil Fuels_)
+  + Eco Cleaning Products used (_good for people who have allergies or who care about the environment_)
++ ***Library*** with ***ALL*** the books you could _ever_ want to read about:
+  + Creativity
+  + Coding
+  + Productivity & Personal Effectiveness
+  + Business
+  + Leadership
+  + (_Postive_) Psychology
+  + All other "Traditional Academic Subjects"
++ **Garden** where you can grow your own food if you want to!
++ State-of-the-Art **Multi-Media Creative Suite** / **Recording Studio**:
+  + Sound Proof
+  + Instruments available
+  + Complete recording equipment
+  + Dedicated Mixing/Editing computers
++ Games Room with:
+  + Pingpong
+  + Foosball
+  + ALL Consoles and Latest Games (_with the **hard rule** that we **only non-violent** "**multi-player**" games and limited to **1h per week per person** if there is a "queue"_)
```

Comment:
Nope. I would _strongly_ argue that Console games have the _best_ UI of any interface design and puzzle/platform games encourage attention to detail, fast reflexes and creative game play. 
and provided people sign our "House Rules" and they keep track of how much time they are spending on games it can _actually_ be a _good_ thing for the brain.
If we provide the Consoles there's no BS around "Timmy broke my toy..."